src/commands/deployWorkspaceProject/deployWorkspaceProject.ts

```diff
@@ -188,8 +205,9 @@ export async function deployWorkspaceProject(context: IActionContext, item?: Con
     await wizard.execute();
 
     displayNotification(wizardContext);
-    ext.branchDataProvider.refresh();
+    wizardContext.telemetry.properties.revisionMode = wizardContext.containerApp?.revisionsMode;
```

Comment:
We can keep this here, but I reccommend that we also track this whenever we list container apps. I think it'll be helpful to see how many users actually use multiple revision mode to further drive some of our design decisions.